runner/runners/queue.go

```diff
@@ -326,7 +379,9 @@ func (c *QueueController) loop() {
 			// Handle run and abort requests.
 			if !ok {
 				c.reqCh = nil
-				c.cancelTimerCh <- nil
+				for i := 0; i < len(c.filerMap); i++ {
+					c.cancelTimerCh <- nil
```

Comment:
Can't you just close the ch once here?

Reply:
We start N updateTickers in separate goroutines, not sure how if we recv on one the others will get the message?

Reply:
All the goroutines will receive "_, notOK := <-ch" when a channel is closed afaik.

Reply:
Ah yes - thanks